xc/common/cmake/arch_define.cmake

```diff
@@ -135,22 +135,18 @@ function(ADD_XC_ARCH_DEFINE)
         --input /dev/stdin \
         --output /dev/stdout \
         \${PLACE_CONSTR_TOOL_EXTRA_ARGS}"
-    BITSTREAM_EXTENSION frames
-    BIN_EXTENSION bit
-    FASM_TO_BIT ${PRJRAY_DIR}/utils/fasm2frames.py
+    BITSTREAM_EXTENSION bit
+    FASM_TO_BIT ${symbiflow-arch-defs_SOURCE_DIR}/env/conda/envs/symbiflow_arch_def_base/bin/xcfasm
```

Comment:
As a rule, if you are writing a path with "conda" in a CMake file, you are doing it wrong.

In this case, you want to add xcfasm to https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/CMakeLists.txt#L32 and use `${XCFASM}` here.

Reply:
Fixed